Scroller/useThemeScroller.js

```diff
@@ -288,7 +288,7 @@ const useSpottable = (props, instances) => {
 		const
 			{rtl} = props,
 			// For Chrome 85+ or Safari that use negative coordinate system for RTL
-			coordinateCoefficient = rtl && (platform.chrome < 85) ? 1 : -1,
+			coordinateCoefficient = rtl && (platform.chrome >= 85) ? -1 : 1,
```

Comment:
I think we should also check safari as original code was
`			coordinateCoefficient = rtl && (platform.ios || platform.safari || platform.chrome >= 85 || platform.androidChrome >= 85) ? -1 : 1,
`
```suggestion
			coordinateCoefficient = rtl && (platform.chrome >= 85 || platform.browserName === 'safari') ? -1 : 1,
```

Reply:
```suggestion
			coordinateCoefficient = rtl && !(platform.chrome < 85) ? -1 : 1;,
```
What about this? It returns 1 for LTR, and returns -1 RTL except when chrome is under 85.

Reply:
It looks OK to me